Extensions/tweaks.js

```diff
@@ -152,6 +152,11 @@ XKit.extensions.tweaks = new Object({
 			default: false,
 			value: false
 		},
+		"wide_sources": {
+			text: "Increase max width of post sources to avoid shortening",
+			default: true,
+			value: false
```

Comment:
If I was more awake I wouldn't have missed that, oops

(It'd be nice if value wasn't necessary to specify in these areas; does it actually give any benefit?)